packages/runtime/src/runtimes/react/controls/rich-text-v2.tsx

```diff
@@ -0,0 +1,16 @@
+import { ReactNode } from 'react'
+
+import {
+  RichTextV2Control,
+  RichTextV2ControlData,
+  RichTextV2ControlDefinition,
+} from '../../../controls'
+import TextV2 from '../../../components/builtin/TextV2'
+
+export type RichTextV2ControlValue = ReactNode
+
+export type Descriptors = { text?: RichTextV2ControlDefinition }
+
+export function useRichTextV2(data: RichTextV2ControlData, control: RichTextV2Control | null) {
+  return <TextV2 text={data} control={control} />
```

Comment:
I see now where the builtin component is being used. We probably want to build that editable from scratch in this file and avoid bringing any bad assumptions or old patterns from the legacy rich text component